app/build.gradle

```diff
@@ -92,7 +92,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     implementation('com.crashlytics.sdk.android:crashlytics:2.8.0@aar') {
-        transitive = true;
+        transitive = true
```

Comment:
this got me thinking, is this flag even necessary? aren't dependencies transitive by default?

Reply:
I tried to compile once without the transitive statement and ran into issues. I agree with your logic though and maybe this was fixed in gradle 4.5.

Reply:
this has bugged me enough to finally google it https://stackoverflow.com/questions/44902339/whats-the-meaning-of-aar-with-transitive-true
TLDR: simply using `implementation 'com.crashlytics.sdk.android:crashlytics:2.8.0'` should work

Reply:
Doing this causes the build to fail, due to problems resolving Crashlytics dependencies. I think there might be some weird stuff in the fabric gradle plugin that makes this a necessity.

Reply:
oh wait I missed the piece on removing the `@aar`. That does seem to work.